shencha-master/shencha-master/test_api_all.py

```
@@ -1,161 +1,219 @@
+
+
+class ValidityCheckRequest(BaseModel):
+    start_date: str = Field(..., description="起始日期 (YYYY-MM-DD)")
+    end_date: str = Field(..., description="结束日期 (YYYY-MM-DD)")
+    docs: Dict[str, Dict[str, Dict[str, Any]]] = Field(
+        ...,
+        description="已提取的文档结构化信息，包括专利和论文"
+    )
+
+
+async def test_process_files(files: List[UploadFile] = File(...)) -> Dict[str, Dict[str, Any]]:
```

Ruff (0.12.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)